nbterm/nbterm.py

```diff
@@ -30,15 +32,23 @@ def main(
     if run:
         assert no_kernel is not True
         asyncio.run(nb.run_all())
-        if save_path is None:
-            i = nb.nb_path.rfind(".")
-            save_path = nb.nb_path[:i] + "_run" + nb.nb_path[i:]
+        save_path = save_path or default_save_path(notebook_path)
         nb.save(save_path)
         typer.echo(f"Executed notebook has been saved to: {save_path}")
     else:
         nb.show()
```

Comment:
I added a default save path branch for when `notebook_path` is `None` below https://github.com/davidbrochart/nbterm/pull/34/files#diff-c74c57deee18c90ee78cda074de21d112c55e0a589bd91bf5e7bf0fdfd3f0d9eR49 , but reading back over this block, it seems like the intent is that if you don't pass a `notebook_path` you end up in the `nb.show()` branch instead. Could you clarify what your intent is for the various combinations of `notebook_path` and `run` that could be passed?